src/views/Manager/ProgressBar.tsx

```diff
@@ -0,0 +1,62 @@
+import { Box, Card, Col, ProgressBar, Row, Text } from '@impact-market/ui';
+import { currencyFormat } from '../../utils/currencies';
+import React from 'react';
+import useTranslations from '../../libs/Prismic/hooks/useTranslations';
+
+const renderProgressBar = (title: string, label: React.ReactNode, value: number, minValue: number) => {
+    const state = value <= minValue ? { error: true } : { info: true };
+
+    return (
+        <>
+            <Row fLayout="start center" margin={0}>
+                <Col colSize={{ sm: 6, xs: 12 }} padding={0}>
+                    <Text g900 semibold>
+                        {title}
+                    </Text>
+                </Col>
+                <Col colSize={{ sm: 6, xs: 12 }} mt={{ sm: 0, xs: 0.25 }} padding={0} tAlign={{ sm: 'right', xs: 'left' }}>
+                    {label}
+                </Col>
+            </Row>
+            <ProgressBar mt={0.5} progress={value} {...state} />
+        </>
+    );
+}
+
+const ProgressBarComponent: React.FC<{ communityEntity: any, currency: string, language: string }> = props => {
+    const { communityEntity, currency, language } = props;
+    const { t } = useTranslations();
+    const goal = communityEntity?.beneficiaries * communityEntity?.maxClaim || 0;
+
+    const localeCurrency = new Intl.NumberFormat(language, {
+        currency,
+        style: 'currency'
+    });
+        
+    return (
+        <Card mt={1.5} padding={1.5} w="100%">
+            <Box>
+                {renderProgressBar(
```

Comment:
WDYT of having a comp instead!?
Personally, I don't like to run functions inside components 😅